src/blocks/remote-data-html/block.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 3,
+  "name": "remote-data-blocks/remote-html",
```

Comment:
Name mismatch between this field and folder name

Reply:
Fixed them both to be `remote-html`, I don't think "data" was adding anything here.